.env.example

```diff
@@ -1,9 +1,12 @@
 UNICORE__LOG_FORMAT=text
-UNICORE__EVENT_STORE__TYPE=postgres
-UNICORE__EVENT_STORE__CONNECTION_STRING="postgresql://demo_user:demo_pass@cqrs-postgres-db:5432/demo"
+UNICORE__EVENT_STORE__TYPE=in_memory
+# UNICORE__EVENT_STORE__CONNECTION_STRING="postgresql://demo_user:demo_pass@cqrs-postgres-db:5432/demo"
 
-UNICORE__SECRET_MANAGER__STRONGHOLD_PATH="agent_secret_manager/tests/res/test.stronghold"
+UNICORE__URL="http://localhost:3033"
+
+UNICORE__SECRET_MANAGER__STRONGHOLD_PATH="agent_secret_manager/tests/res/temp.stronghold"
+# UNICORE__SECRET_MANAGER__STRONGHOLD_PATH="agent_secret_manager/tests/res/test.stronghold"
```

Comment:
The actual values of the commented and uncommented `UNICORE__SECRET_MANAGER__STRONGHOLD_PATH` variable are not
duplicates. But I now moved the commented version down together with its corresponding secret manager variables so that
it is more clear that they belong together.